core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -648,6 +648,9 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       _ <- createWdsAppInWorkspace(workspaceId, parentContext, None, workspaceRequest.attributes)
 
     } yield savedWorkspace).recoverWith {
+      case r: RawlsExceptionWithErrorReport if r.errorReport.statusCode.contains(StatusCodes.Conflict) =>
```

Comment:
Thrown by https://github.com/broadinstitute/rawls/blob/bae3a702f5f9e321b25b607f95abe783a62a9d49/core/src/main/scala/org/broadinstitute/dsde/rawls/util/WorkspaceSupport.scala#L279

Reply:
Thanks for the pointer! I will add a test case for that.

Reply:
I'd like this to be closer/more tightly scoped to the call site. But I don't see a way to do that without refactoring the entire function (which I'm sure is way more than what you want to do right now).

Which is to say: LGTM.